### PR TITLE
63 finalize logging system

### PR DESCRIPTION
**Pull request by Paldana99**

### CODE REVIEW

File: scripts/amqp_listen.py [hunk 1]
```diff
@@ -34,13 +35,13 @@ async def main() -> None:
...
-                    print(json.dumps(parsed_json, indent=4))
-                    print("".center(80, "="))
-                    print("".center(80, "*"))
+                    logger.info("".center(80, "*"))
+                    logger.info("".center(80, "="))
+                    logger.info("message received".center(80, "="))
+                    logger.info("".center(80, "="))
+                    logger.info(json.dumps(parsed_json, indent=4))
+                    logger.info("".center(80, "="))
+                    logger.info("".center(80, "*"))
```
Comment (jdp1ps) [inline scripts/amqp_listen.py:44]: The script is not part of the application and should not use the logging system : it's just a fake client.

**Review by jdp1ps — APPROVED** [review]

File: tests/conftest.py [hunk 2]
```diff
@@ -65,23 +65,3 @@ async def fixture_async_session() -> AsyncGenerator[AsyncSession, None]:
...
-    """
-    handler_id = logger.add(
-        caplog.handler,
-        format="{message}",
-        level=0,
-        filter=lambda record: record["level"].no >= caplog.handler.level,
-        enqueue=True,
-    )
-    yield caplog
-    logger.remove(handler_id)
```
Comment (jdp1ps) [inline tests/conftest.py:87]: @Paldana99 Why did you disable the fixture ?

Comment (Paldana99) [reply tests/conftest.py:87]: It cause all test to fail. Due that now the config is made directly in the svp_harvester

Comment (jdp1ps) [reply tests/conftest.py:87]: @Paldana99 What if we need caplog ? https://loguru.readthedocs.io/en/stable/resources/migration.html#replacing-caplog-fixture-from-pytest-library

Comment (Paldana99) [reply tests/conftest.py:87]: Now is working properly, i restore the fixture. And added a test with a  caplog assert.

**Review by jdp1ps — APPROVED** [review]

**Pull request merged**